packages/internal/bridge/bridge-sample-app/src/lib/helpers.ts

```diff
@@ -0,0 +1,78 @@
+import { CHILD_ERC20, CHILD_ERC20_BRIDGE, ERC20, ROOT_ERC20_BRIDGE_FLOW_RATE } from "@imtbl/bridge-sdk";
+import { providers, ethers } from "ethers";
+
+export function delay(time: number) {
+    return new Promise(resolve => setTimeout(resolve, time));
+}
+
+export async function waitForReceipt(txHash: string, provider: providers.JsonRpcProvider) {
+    let receipt;
+    console.log("Wait until receipt... tx hash: ", txHash);
+    let attempt = 0;
+    while (receipt == null) {
+        attempt++;
+        console.log('waiting for receipt attempt: ', attempt);
+        receipt = await provider.getTransactionReceipt(txHash)
+        await delay(1000);
```

Comment:
this delay might be unnecessary if `getTransactionReceipt()` succeeded. suggest checking the state of `receipt` before applying.